microk8s-resources/wrappers/microk8s-enable.wrapper

```diff
@@ -35,17 +35,21 @@ for addon in "$@"; do
   fi
   
   # If there is a script to execute for the $action run the script and ignore any yamls
-  if [ -f "${SNAP}/actions/enable.$action.sh" ]; then
-    "${SNAP}/actions/enable.$action.sh" "${arguments}"
-    enable_result="$?"
-    if [ "$enable_result" = "0" ]
-    then
-      result=0
-    else
-      echo "Failed to enable $action"
-      exit 1
+  for filepath in "${SNAP}/actions/enable.$action"*; do
+    if [ -f ${filepath} ]; then
+      "${filepath}" "${arguments}"
+      enable_result="$?"
+      if [ "$enable_result" = "0" ]; then
+        result=0
+        break
+      else
+        echo "Failed to enable $action"
+        exit 1
+      fi
     fi
-  elif [ -f "${SNAP}/actions/$action.yaml" ]; then
+  done
+
+  if [ -f "${SNAP}/actions/$action.yaml" ]; then
```

Comment:
This is a difference in behavior.

If we have both `enable.foo.sh` and `foo.yaml` files the `foo.yaml` in skipped. With this PR we run the script and then we also apply the manifest. This is not what we want.

Consider the case on the ingress addon. There is an ingress.yaml that serves as a template, and the enable.ingress.sh parses the template yaml and applies it. We do not want to apply the template yaml.

Reply:
I see how that is changed. However, I have trouble understanding in which case the yaml would be applied then. In the ingress example, the code will always find first the enable.ingress action, so it would never run the yaml manifest. When is this actually used?

Reply:
Ok, so from the whole list, only metrics-server.yaml does not have an "enable.metrics-server.sh" equivalent, so that would be the only use case so far for that chunk of code

Reply:
New PR pushed, it should resolve this.